src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/xcframework-installation.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: XCFramework manual installation
+metaDescription: How to instrument to your iOS applications by downloading the XCFramework.
+freshnessValidatedDate: 2024-7-16
+---
+
+We recommend that you use our [guided install](https://onenr.io/0qwLv87gkj5) to set up iOS monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic iOS agent by downloading the XCFramework.
+
+## Manually add the New Relic XCFramework to your project
+
+<Steps>
+    <Step>
+Download the latest iOS agent (XCFramework) from our [iOS agent release notes](/docs/release-notes/mobile-release-notes/ios-release-notes).        
+    </Step>
+    <Step>
+Navigate to where the iOS agent was downloaded and unzip it.        
+    </Step>
+    <Step>In your Xcode project, add the unzipped agent:
+
+1. Right-click on your Project navigator and click **Add Files to `[YourProjectName]`**.
+2. Select the `NewRelic.xcframework` directory and click **Add**.        
+    </Step>
+    <Step>
+Embed the iOS agent:
+1. In the Project navigator, open your project and target settings.
+2. Click on the target to which you want to add the agent.
```

Comment:
@mbruin-NR How do they know/decide which target to add the agent? I think we should add more on this, like if we have recommendations on where to add it

Reply:
_Targets_ is a key word in Xcode development. It is the umbrella under which your app is constructed in you project. If the reader has used Xcode to make an iOS, tvOS, watchOS project I think they will understand the sentence.